package.json

```diff
@@ -17,6 +17,9 @@
     "postmerge": "yarn install",
     "precommit": "git status --porcelain | awk 'match($1, \"[ACM]\"){print $2}' | grep '[js|jsx]$' | xargs eslint",
     "storybook": "start-storybook -p 9001 -c .storybook",
+    "storybook-static": "build-storybook -c .storybook -o .out",
+    "gh-pages": "gh-pages -d .out -b gh-pages",
+    "deploy": "yarn run storybook-static && yarn run gh-pages",
```

Comment:
`deploy` is a little ambiguous for this repo, because it's not like the rest of Meetup's deployments - maybe `publish` or something?